### ISSUE TITLE
Cant Upload SVG Files | /elementor/core/files/assets/svg/svg-handler.php::656

### ISSUE THREAD

**Issue by maurocarreraok** [elementor/elementor#16088]
## Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

## Description

Cant upload SVG Files.

![image](https://user-images.githubusercontent.com/68023943/130236744-deb5d6de-5fe8-4f74-b048-fb1156e06040.png)

![image](https://user-images.githubusercontent.com/68023943/130236847-5849b752-9fe3-4419-aeab-b1e29473f260.png)

## Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a default WordPress theme active.
- [X] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```

					
== Server Environment ==
	Operating System: Linux
	Software: nginx/1.20.1
	MySQL version: MariaDB Server v10.6.4
	PHP Version: 7.3.29
	PHP Max Input Vars: 3000
	PHP Max Post Size: 8M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 5.8
	Site URL: https://reports..com
	Home URL: https://reports..com
	WP Multisite: No
	Max Upload Size: 8 MB
	Memory limit: 2048M
	Permalink Structure: /%postname%/
	Language: en-US
	Timezone: America/New_York
	Debug Mode: Inactive

== Theme ==
	Name:  Theme
	Version: 2.0.0
	Author: 
	Child Theme: Yes
	Parent Theme Name: Hello Elementor
	Parent Theme Version: 2.4.1
	Parent Theme Author: Elementor Team

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/92.0.4515.159 Safari/537.36

== Active Plugins ==
	Crocoblock Wizard
		Version: 1.1.0
		Author: Crocoblock

	Elementor
		Version: 3.4.2
		Author: Elementor.com

	Elementor Pro
		Version: 3.3.7
		Author: Elementor.com

	JetEngine
		Version: 2.8.10
		Author: Crocoblock

	TranslatePress - Multilingual
		Version: 2.0.8
		Author: Cozmoslabs, Razvan Mocanu, Madalin Ungureanu, Cristophor Hurduban

	WPSHAPERE
		Version: 6.1.13
		Author: AcmeeDesign Softwares and Solutions


== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatible

== Elementor Pro - Compatibility Tag ==
	


== Elements Usage ==
	
	header : 2
		button : 2
		nav-menu : 2
		shortcode : 2
		theme-site-logo : 2


== Log ==
	
Log: showing 6 of 62021-08-20 07:55:11 [info] elementor::elementor_updater Started 
2021-08-20 07:55:11 [info] Elementor/Upgrades - _on_each_version Start  
2021-08-20 07:55:11 [info] Elementor/Upgrades - _on_each_version Finished 
2021-08-20 07:55:11 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.3.1',
  'to' => '3.4.2',
)]
2021-08-20 07:55:11 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.3.1',
  'to' => '3.4.2',
)]
2021-08-20 08:11:04 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.3.1',
  'to' => '3.3.7',
)]

PHP: showing 18 of 18PHP: 2021-08-20 07:55:13 [error X 1][/var/www/report/wp-content/plugins/elementor/core/wp-api.php::26] Uncaught Error: Call to undefined function Elementor\Core\get_plugins() in /var/www/report/wp-content/plugins/elementor/core/wp-api.php:26
Stack trace:
#0 /var/www/report/wp-content/plugins/elementor/core/wp-api.php(36): Elementor\Core\Wp_Api->get_plugins()
#1 /var/www/report/wp-content/plugins/elementor/modules/system-info/reporters/plugins.php(44): Elementor\Core\Wp_Api->get_active_plugins()
#2 /var/www/report/wp-content/plugins/elementor/modules/system-info/reporters/plugins.php(75): Elementor\Modules\System_Info\Reporters\Plugins->get_plugins()
#3 /var/www/report/wp-content/plugins/elementor/modules/system-info/module.php(298): Elementor\Modules\System_Info\Reporters\Plugins->is_enabled()
#4 /var/www/report/wp-content/plugins/elementor/modules/system-info/module.php(255): Elementor\Modules\System_Info\Module->create_reporter(Array)
#5 /var/www/report/wp-content/plugins/elementor/includes/tracker.php(228): Elementor\Modules\System_Info\Module->load_re [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2021-08-20 08:05:43 [notice X 2][/var/www/report/wp-content/plugins/elementor/core/common/modules/connect/apps/base-app.php::587] Trying to get property 'auth_secret' of non-object [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2021-08-20 08:40:08 [error X 1][/var/www/report/wp-content/plugins/elementor/core/files/assets/svg/svg-handler.php::656] Uncaught Error: Call to a member function attributes() on bool in /var/www/report/wp-content/plugins/elementor/core/files/assets/svg/svg-handler.php:656
Stack trace:
#0 /var/www/report/wp-includes/class-wp-hook.php(303): Elementor\Core\Files\Assets\Svg\Svg_Handler->set_svg_meta_data(Array, 88)
#1 /var/www/report/wp-includes/plugin.php(189): WP_Hook->apply_filters(Array, Array)
#2 /var/www/report/wp-includes/post.php(6366): apply_filters('wp_update_attac...', Array, 88)
#3 /var/www/report/wp-admin/includes/media.php(421): wp_update_attachment_metadata(88, Array)
#4 /var/www/report/wp-admin/includes/ajax-actions.php(2549): media_handle_upload('async-upload', '11', Array)
#5 /var/www/report/wp-admin/async-upload.php(33): wp_ajax_upload_attachment()
#6 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2021-08-20 08:40:09 [error X 5][/var/www/report/wp-content/plugins/elementor/core/files/assets/svg/svg-handler.php::656] Uncaught Error: Call to a member function attributes() on bool in /var/www/report/wp-content/plugins/elementor/core/files/assets/svg/svg-handler.php:656
Stack trace:
#0 /var/www/report/wp-includes/class-wp-hook.php(303): Elementor\Core\Files\Assets\Svg\Svg_Handler->set_svg_meta_data(Array, 88)
#1 /var/www/report/wp-includes/plugin.php(189): WP_Hook->apply_filters(Array, Array)
#2 /var/www/report/wp-includes/post.php(6366): apply_filters('wp_update_attac...', Array, 88)
#3 /var/www/report/wp-admin/includes/image.php(178): wp_update_attachment_metadata(88, Array)
#4 /var/www/report/wp-admin/includes/ajax-actions.php(2459): wp_update_image_subsizes(88)
#5 /var/www/report/wp-includes/class-wp-hook.php(303): wp_ajax_media_create_image_subsizes('')
#6 /var/www/report/wp-includes/class-wp-hook.php(327): WP_Hook->apply_filters('', Array)
#7 /var/www/report/wp-includes/plugin.php(470): WP_Hook->do_action(Array)
#8 /var/www/report/wp-admin/ [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2021-08-20 08:40:29 [error X 1][/var/www/report/wp-content/plugins/elementor/core/files/assets/svg/svg-handler.php::656] Uncaught Error: Call to a member function attributes() on bool in /var/www/report/wp-content/plugins/elementor/core/files/assets/svg/svg-handler.php:656
Stack trace:
#0 /var/www/report/wp-includes/class-wp-hook.php(303): Elementor\Core\Files\Assets\Svg\Svg_Handler->set_svg_meta_data(Array, 89)
#1 /var/www/report/wp-includes/plugin.php(189): WP_Hook->apply_filters(Array, Array)
#2 /var/www/report/wp-includes/post.php(6366): apply_filters('wp_update_attac...', Array, 89)
#3 /var/www/report/wp-admin/includes/media.php(421): wp_update_attachment_metadata(89, Array)
#4 /var/www/report/wp-admin/includes/ajax-actions.php(2549): media_handle_upload('async-upload', '11', Array)
#5 /var/www/report/wp-admin/async-upload.php(33): wp_ajax_upload_attachment()
#6 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2021-08-20 08:40:30 [error X 5][/var/www/report/wp-content/plugins/elementor/core/files/assets/svg/svg-handler.php::656] Uncaught Error: Call to a member function attributes() on bool in /var/www/report/wp-content/plugins/elementor/core/files/assets/svg/svg-handler.php:656
Stack trace:
#0 /var/www/report/wp-includes/class-wp-hook.php(303): Elementor\Core\Files\Assets\Svg\Svg_Handler->set_svg_meta_data(Array, 89)
#1 /var/www/report/wp-includes/plugin.php(189): WP_Hook->apply_filters(Array, Array)
#2 /var/www/report/wp-includes/post.php(6366): apply_filters('wp_update_attac...', Array, 89)
#3 /var/www/report/wp-admin/includes/image.php(178): wp_update_attachment_metadata(89, Array)
#4 /var/www/report/wp-admin/includes/ajax-actions.php(2459): wp_update_image_subsizes(89)
#5 /var/www/report/wp-includes/class-wp-hook.php(303): wp_ajax_media_create_image_subsizes('')
#6 /var/www/report/wp-includes/class-wp-hook.php(327): WP_Hook->apply_filters('', Array)
#7 /var/www/report/wp-includes/plugin.php(470): WP_Hook->do_action(Array)
#8 /var/www/report/wp-admin/ [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2021-08-20 08:40:48 [error X 1][/var/www/report/wp-content/plugins/elementor/core/files/assets/svg/svg-handler.php::656] Uncaught Error: Call to a member function attributes() on bool in /var/www/report/wp-content/plugins/elementor/core/files/assets/svg/svg-handler.php:656
Stack trace:
#0 /var/www/report/wp-includes/class-wp-hook.php(303): Elementor\Core\Files\Assets\Svg\Svg_Handler->set_svg_meta_data(Array, 90)
#1 /var/www/report/wp-includes/plugin.php(189): WP_Hook->apply_filters(Array, Array)
#2 /var/www/report/wp-includes/post.php(6366): apply_filters('wp_update_attac...', Array, 90)
#3 /var/www/report/wp-admin/includes/media.php(421): wp_update_attachment_metadata(90, Array)
#4 /var/www/report/wp-admin/includes/ajax-actions.php(2549): media_handle_upload('async-upload', '66', Array)
#5 /var/www/report/wp-admin/async-upload.php(33): wp_ajax_upload_attachment()
#6 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2021-08-20 08:40:48 [error X 5][/var/www/report/wp-content/plugins/elementor/core/files/assets/svg/svg-handler.php::656] Uncaught Error: Call to a member function attributes() on bool in /var/www/report/wp-content/plugins/elementor/core/files/assets/svg/svg-handler.php:656
Stack trace:
#0 /var/www/report/wp-includes/class-wp-hook.php(303): Elementor\Core\Files\Assets\Svg\Svg_Handler->set_svg_meta_data(Array, 90)
#1 /var/www/report/wp-includes/plugin.php(189): WP_Hook->apply_filters(Array, Array)
#2 /var/www/report/wp-includes/post.php(6366): apply_filters('wp_update_attac...', Array, 90)
#3 /var/www/report/wp-admin/includes/image.php(178): wp_update_attachment_metadata(90, Array)
#4 /var/www/report/wp-admin/includes/ajax-actions.php(2459): wp_update_image_subsizes(90)
#5 /var/www/report/wp-includes/class-wp-hook.php(303): wp_ajax_media_create_image_subsizes('')
#6 /var/www/report/wp-includes/class-wp-hook.php(327): WP_Hook->apply_filters('', Array)
#7 /var/www/report/wp-includes/plugin.php(470): WP_Hook->do_action(Array)
#8 /var/www/report/wp-admin/ [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2021-08-20 08:41:23 [error X 1][/var/www/report/wp-content/plugins/elementor/core/files/assets/svg/svg-handler.php::656] Uncaught Error: Call to a member function attributes() on bool in /var/www/report/wp-content/plugins/elementor/core/files/assets/svg/svg-handler.php:656
Stack trace:
#0 /var/www/report/wp-includes/class-wp-hook.php(303): Elementor\Core\Files\Assets\Svg\Svg_Handler->set_svg_meta_data(Array, 91)
#1 /var/www/report/wp-includes/plugin.php(189): WP_Hook->apply_filters(Array, Array)
#2 /var/www/report/wp-includes/post.php(6366): apply_filters('wp_update_attac...', Array, 91)
#3 /var/www/report/wp-admin/includes/media.php(421): wp_update_attachment_metadata(91, Array)
#4 /var/www/report/wp-admin/async-upload.php(114): media_handle_upload('async-upload', 0)
#5 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2021-08-20 08:41:23 [error X 5][/var/www/report/wp-content/plugins/elementor/core/files/assets/svg/svg-handler.php::656] Uncaught Error: Call to a member function attributes() on bool in /var/www/report/wp-content/plugins/elementor/core/files/assets/svg/svg-handler.php:656
Stack trace:
#0 /var/www/report/wp-includes/class-wp-hook.php(303): Elementor\Core\Files\Assets\Svg\Svg_Handler->set_svg_meta_data(Array, 91)
#1 /var/www/report/wp-includes/plugin.php(189): WP_Hook->apply_filters(Array, Array)
#2 /var/www/report/wp-includes/post.php(6366): apply_filters('wp_update_attac...', Array, 91)
#3 /var/www/report/wp-admin/includes/image.php(178): wp_update_attachment_metadata(91, Array)
#4 /var/www/report/wp-admin/includes/ajax-actions.php(2459): wp_update_image_subsizes(91)
#5 /var/www/report/wp-includes/class-wp-hook.php(303): wp_ajax_media_create_image_subsizes('')
#6 /var/www/report/wp-includes/class-wp-hook.php(327): WP_Hook->apply_filters('', Array)
#7 /var/www/report/wp-includes/plugin.php(470): WP_Hook->do_action(Array)
#8 /var/www/report/wp-admin/ [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2021-08-20 08:44:14 [error X 1][/var/www/report/wp-content/plugins/elementor/core/files/assets/svg/svg-handler.php::656] Uncaught Error: Call to a member function attributes() on bool in /var/www/report/wp-content/plugins/elementor/core/files/assets/svg/svg-handler.php:656
Stack trace:
#0 /var/www/report/wp-includes/class-wp-hook.php(303): Elementor\Core\Files\Assets\Svg\Svg_Handler->set_svg_meta_data(Array, 93)
#1 /var/www/report/wp-includes/plugin.php(189): WP_Hook->apply_filters(Array, Array)
#2 /var/www/report/wp-includes/post.php(6366): apply_filters('wp_update_attac...', Array, 93)
#3 /var/www/report/wp-admin/includes/media.php(421): wp_update_attachment_metadata(93, Array)
#4 /var/www/report/wp-admin/async-upload.php(114): media_handle_upload('async-upload', 0)
#5 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2021-08-20 08:44:15 [error X 5][/var/www/report/wp-content/plugins/elementor/core/files/assets/svg/svg-handler.php::656] Uncaught Error: Call to a member function attributes() on bool in /var/www/report/wp-content/plugins/elementor/core/files/assets/svg/svg-handler.php:656
Stack trace:
#0 /var/www/report/wp-includes/class-wp-hook.php(303): Elementor\Core\Files\Assets\Svg\Svg_Handler->set_svg_meta_data(Array, 93)
#1 /var/www/report/wp-includes/plugin.php(189): WP_Hook->apply_filters(Array, Array)
#2 /var/www/report/wp-includes/post.php(6366): apply_filters('wp_update_attac...', Array, 93)
#3 /var/www/report/wp-admin/includes/image.php(178): wp_update_attachment_metadata(93, Array)
#4 /var/www/report/wp-admin/includes/ajax-actions.php(2459): wp_update_image_subsizes(93)
#5 /var/www/report/wp-includes/class-wp-hook.php(303): wp_ajax_media_create_image_subsizes('')
#6 /var/www/report/wp-includes/class-wp-hook.php(327): WP_Hook->apply_filters('', Array)
#7 /var/www/report/wp-includes/plugin.php(470): WP_Hook->do_action(Array)
#8 /var/www/report/wp-admin/ [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2021-08-20 08:44:34 [error X 1][/var/www/report/wp-content/plugins/elementor/core/files/assets/svg/svg-handler.php::656] Uncaught Error: Call to a member function attributes() on bool in /var/www/report/wp-content/plugins/elementor/core/files/assets/svg/svg-handler.php:656
Stack trace:
#0 /var/www/report/wp-includes/class-wp-hook.php(303): Elementor\Core\Files\Assets\Svg\Svg_Handler->set_svg_meta_data(Array, 94)
#1 /var/www/report/wp-includes/plugin.php(189): WP_Hook->apply_filters(Array, Array)
#2 /var/www/report/wp-includes/post.php(6366): apply_filters('wp_update_attac...', Array, 94)
#3 /var/www/report/wp-admin/includes/media.php(421): wp_update_attachment_metadata(94, Array)
#4 /var/www/report/wp-admin/includes/ajax-actions.php(2549): media_handle_upload('async-upload', '66', Array)
#5 /var/www/report/wp-admin/async-upload.php(33): wp_ajax_upload_attachment()
#6 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2021-08-20 08:44:35 [error X 5][/var/www/report/wp-content/plugins/elementor/core/files/assets/svg/svg-handler.php::656] Uncaught Error: Call to a member function attributes() on bool in /var/www/report/wp-content/plugins/elementor/core/files/assets/svg/svg-handler.php:656
Stack trace:
#0 /var/www/report/wp-includes/class-wp-hook.php(303): Elementor\Core\Files\Assets\Svg\Svg_Handler->set_svg_meta_data(Array, 94)
#1 /var/www/report/wp-includes/plugin.php(189): WP_Hook->apply_filters(Array, Array)
#2 /var/www/report/wp-includes/post.php(6366): apply_filters('wp_update_attac...', Array, 94)
#3 /var/www/report/wp-admin/includes/image.php(178): wp_update_attachment_metadata(94, Array)
#4 /var/www/report/wp-admin/includes/ajax-actions.php(2459): wp_update_image_subsizes(94)
#5 /var/www/report/wp-includes/class-wp-hook.php(303): wp_ajax_media_create_image_subsizes('')
#6 /var/www/report/wp-includes/class-wp-hook.php(327): WP_Hook->apply_filters('', Array)
#7 /var/www/report/wp-includes/plugin.php(470): WP_Hook->do_action(Array)
#8 /var/www/report/wp-admin/ [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2021-08-20 08:45:09 [error X 1][/var/www/report/wp-content/plugins/elementor/core/files/assets/svg/svg-handler.php::656] Uncaught Error: Call to a member function attributes() on bool in /var/www/report/wp-content/plugins/elementor/core/files/assets/svg/svg-handler.php:656
Stack trace:
#0 /var/www/report/wp-includes/class-wp-hook.php(303): Elementor\Core\Files\Assets\Svg\Svg_Handler->set_svg_meta_data(Array, 95)
#1 /var/www/report/wp-includes/plugin.php(189): WP_Hook->apply_filters(Array, Array)
#2 /var/www/report/wp-includes/post.php(6366): apply_filters('wp_update_attac...', Array, 95)
#3 /var/www/report/wp-admin/includes/media.php(421): wp_update_attachment_metadata(95, Array)
#4 /var/www/report/wp-admin/includes/ajax-actions.php(2549): media_handle_upload('async-upload', '66', Array)
#5 /var/www/report/wp-admin/async-upload.php(33): wp_ajax_upload_attachment()
#6 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2021-08-20 08:45:09 [error X 5][/var/www/report/wp-content/plugins/elementor/core/files/assets/svg/svg-handler.php::656] Uncaught Error: Call to a member function attributes() on bool in /var/www/report/wp-content/plugins/elementor/core/files/assets/svg/svg-handler.php:656
Stack trace:
#0 /var/www/report/wp-includes/class-wp-hook.php(303): Elementor\Core\Files\Assets\Svg\Svg_Handler->set_svg_meta_data(Array, 95)
#1 /var/www/report/wp-includes/plugin.php(189): WP_Hook->apply_filters(Array, Array)
#2 /var/www/report/wp-includes/post.php(6366): apply_filters('wp_update_attac...', Array, 95)
#3 /var/www/report/wp-admin/includes/image.php(178): wp_update_attachment_metadata(95, Array)
#4 /var/www/report/wp-admin/includes/ajax-actions.php(2459): wp_update_image_subsizes(95)
#5 /var/www/report/wp-includes/class-wp-hook.php(303): wp_ajax_media_create_image_subsizes('')
#6 /var/www/report/wp-includes/class-wp-hook.php(327): WP_Hook->apply_filters('', Array)
#7 /var/www/report/wp-includes/plugin.php(470): WP_Hook->do_action(Array)
#8 /var/www/report/wp-admin/ [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2021-08-20 08:45:37 [error X 1][/var/www/report/wp-content/plugins/elementor/core/files/assets/svg/svg-handler.php::656] Uncaught Error: Call to a member function attributes() on bool in /var/www/report/wp-content/plugins/elementor/core/files/assets/svg/svg-handler.php:656
Stack trace:
#0 /var/www/report/wp-includes/class-wp-hook.php(303): Elementor\Core\Files\Assets\Svg\Svg_Handler->set_svg_meta_data(Array, 96)
#1 /var/www/report/wp-includes/plugin.php(189): WP_Hook->apply_filters(Array, Array)
#2 /var/www/report/wp-includes/post.php(6366): apply_filters('wp_update_attac...', Array, 96)
#3 /var/www/report/wp-admin/includes/media.php(421): wp_update_attachment_metadata(96, Array)
#4 /var/www/report/wp-admin/includes/ajax-actions.php(2549): media_handle_upload('async-upload', '66', Array)
#5 /var/www/report/wp-admin/async-upload.php(33): wp_ajax_upload_attachment()
#6 {main}
  thrown [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2021-08-20 08:45:38 [error X 5][/var/www/report/wp-content/plugins/elementor/core/files/assets/svg/svg-handler.php::656] Uncaught Error: Call to a member function attributes() on bool in /var/www/report/wp-content/plugins/elementor/core/files/assets/svg/svg-handler.php:656
Stack trace:
#0 /var/www/report/wp-includes/class-wp-hook.php(303): Elementor\Core\Files\Assets\Svg\Svg_Handler->set_svg_meta_data(Array, 96)
#1 /var/www/report/wp-includes/plugin.php(189): WP_Hook->apply_filters(Array, Array)
#2 /var/www/report/wp-includes/post.php(6366): apply_filters('wp_update_attac...', Array, 96)
#3 /var/www/report/wp-admin/includes/image.php(178): wp_update_attachment_metadata(96, Array)
#4 /var/www/report/wp-admin/includes/ajax-actions.php(2459): wp_update_image_subsizes(96)
#5 /var/www/report/wp-includes/class-wp-hook.php(303): wp_ajax_media_create_image_subsizes('')
#6 /var/www/report/wp-includes/class-wp-hook.php(327): WP_Hook->apply_filters('', Array)
#7 /var/www/report/wp-includes/plugin.php(470): WP_Hook->do_action(Array)
#8 /var/www/report/wp-admin/ [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]


				

```
</details>

**Comment by mcrowell** [comment]
+1 Same error on my end.

**Comment by shilo-ey** [comment]
Thank you for reporting.

We will handle this issue soon and release a patch.

I merged this issue with #16084, which is on the same subject so we could deal with it more efficiently. 
Please feel free to continue the conversation there.

Thanks for your feedback!

**Comment by shilo-ey** [comment]
This issue has been resolved in Elementor v3.4.3

Feel free to update

Thanks!
